packages/hash/realtime/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hashintel/hash-realtime",
-  "version": "0.1.0",
+  "version": "0.0.0",
```

Comment:
Why are our workspace packages changed to be 0.0.0 across the board? I assume this relates to `check-dependency-version-consistency`

Reply:
I mentioned this in PR comment 👀

Reply:
My bad, missed that

Reply:
No worries! Shall we give it a go?